src/style.css

```diff
@@ -0,0 +1,62 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  box-sizing: border-box;
+  color: #3e4e61;
+  font-weight: 300;
+}
+
+.container {
+  margin: 20px 0;
+}
+
+.searchBar {
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+  outline: none;
+}
+
+.searchBar.big {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  padding: 23px 20px 23px 62px;
+  background-position: left 26px bottom 26px;
+  background-size: 19px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.searchBar.small {
+  width: 100%;
+  height: 42px;
+  size: 14px;
+  padding: 12px 20px 11px 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
```

Comment:
Set image as a background for ::before pseudo-element for input (now I don't see the image when I click on the input)